utils/polus-imagej-util/README.md

```diff
@@ -11,14 +11,49 @@ In addition to the above criteria, at this time only the required inputs of the
 
 ## How to use
 
-1. Install cookiecutter: `pip install cookiecutter`
-2. Clone `polus-plugins` and change to the polus-plugins directory
-3. Modify `generate.py` to include author information
-4. Run `generate.py` or modify `generate.py` to generate desired plugins
-
+1. Install pipeline dependencies (see below)
+2. Clone `polus-plugins` and change to the `polus-plugins` directory
+3. Run `utils/polus-imagej-util/classes/populate.py` to gather imagej op info
+4. Open and run `utils/polus-imagej-util/imagej_ui.ipynb`
+5. Select a plugin and make desired edits to fields (varaible names, variable descriptions, version, etc.)
+6. Press `Save File` button at bottom of dashboard, this saves the user's edits to the plugin's cookiecutter template file
+7. Press `Generate Plugin` button at bottom of dashboard, this will create the plugin's directory in `polus-plugins`
+8. Press `Build Docker` button at bottom of dashboard, this will create a docker image for the plugin
+9. The plugin's docker image can now be mounted and run in WIPP.
 
 ** NOTE: ** Do not modify `project_slug`. This is automatically generated from the name of the plugin. If the imagej plugin is called "Imagej Image Integral", then the plugin directory and docker container will have the name `polus-imagej-image-integral-plugin`.
 
+### Installing Dependencies
+
+To run `populate.py` or run a plugin outside of WIPP the following dependencies need to be installed:
+
+- `pyimagej=1.0.0`
+- `openjdk=8`
+- `scyjava=1.1.0`
+- `jpype1=1.2.1`
+- `imglyb=1.0.0`
+- `bfio[all]==2.1.9`
```

Comment:
This can be simplified by telling people to install openjdk 8 (provide a link and/or an `apt-get` command for installation) and then updating the `requirements.txt` file. I think one exists, but if it doesn't then create it and put this information into it. That way you can just do `pip install -r requirements.txt`, and then you don't have to update the README if you ever update the dependencies, just the requirements file.

Reply:
This has been updated in the `README.md` and a `requirements.txt` file has been added for the pipeline. There was a `requirements.txt` for generated plugins but not for the pipeline.